libraries/src/Component/ComponentHelper.php

```diff
@@ -217,31 +220,32 @@ public static function filterText($text)
 						$customListAttributes = array_merge($customListAttributes, $tempAttributes);
 					}
 				}
-				elseif ($filterType === 'WL')
+				// "WL" is deprecated in Joomla! 4, will be removed in Joomla! 5
+				elseif (in_array($filterType, ['WL', 'AL']))
 				{
-					$whiteList           = true;
-					$whiteListTags       = array_merge($whiteListTags, $tempTags);
-					$whiteListAttributes = array_merge($whiteListAttributes, $tempAttributes);
+					$allowList       = true;
+					$allowedTags       = array_merge($allowedTags, $tempTags);
+					$allowedAttributes = array_merge($allowedAttributes, $tempAttributes);
 				}
 			}
 		}
 
-		// Remove duplicates before processing (because the blacklist uses both sets of arrays).
-		$blackListTags        = array_unique($blackListTags);
-		$blackListAttributes  = array_unique($blackListAttributes);
+		// Remove duplicates before processing (because the disallowlist uses both sets of arrays).
```

Comment:
```suggestion
		// Remove duplicates before processing (because the Disallow List uses both sets of arrays).
```
Formalise the wording to show as separate words.